modules/spring-config/pom.xml

```diff
@@ -13,6 +13,42 @@
         Mule Builder for use with Spring 2.X Namespace based XML configuration.
     </description>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>2.4.3</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <!-- Build a jar with cglib 3.2.2 shaded for spring -->
```

Comment:
I would remove the version from here as it will be probably out of sync with the version used